README.md

```diff
@@ -163,14 +171,14 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
 
 10. Navigate to the `/ODIM/odim-controller/scripts` directory on the deployment node.
 
-    ```
-	$ cd ~/ODIM/odim-controller/scripts
-    ```
-    
-12. Open the `kube_deploy_nodes.yaml` file.
+   ```
+   $ cd ~/ODIM/odim-controller/scripts
+   ```
+
+11. Open the `kube_deploy_nodes.yaml` file.
 
          $ vi kube_deploy_nodes.yaml
-    
+
 12. Update the following parameters in the `kube_deploy_nodes.yaml` file to their corresponding values: 
```

Comment:
add below as 13th step:
Move odimra_kafka_client.key and odimra_kafka_client.crt stored in the path specified for odimCertsPath to a different folder.

Update odimra-secrets:

$ python3 odim-controller.py --config \
 /home/${USER}/ODIM/odim-controller/scripts/kube_deploy_nodes.yaml --upgrade odimra-secret